<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Traitement csv</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Gill Sans Ultra Bold Condensed&quot;;
color: rgb(255, 0, 255);</string>
   </property>
   <property name="text">
    <string>Fichier csv</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNomFichier">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCharger">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);
font: 75 12pt &quot;Arial&quot;;
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Charger</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Gill Sans Ultra Bold Condensed&quot;;
color: rgb(255, 0, 255);</string>
   </property>
   <property name="text">
    <string>Filtrer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFiltrer">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Gill Sans Ultra Bold Condensed&quot;;
color: rgb(255, 0, 255);</string>
   </property>
   <property name="text">
    <string>Trier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCroissant">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
   </property>
   <property name="text">
    <string> ˄</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDecroissant">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
   </property>
   <property name="text">
    <string> ˅</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblDonnees">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>581</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEnregistrer">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>enregistrer.png</normaloff>enregistrer.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
